src/extensions/LinkSiteTree.php

```diff
@@ -61,7 +64,33 @@ public function updateCMSFields(FieldList $fields)
     {
         $owner = $this->owner;
         $config = $owner->config();
-        $sitetree_field_label = $config->get('sitetree_field_label') ? : 'MenuTitle';
+        $sitetree_field_label = $config->get('sitetree_field_label') ?: 'MenuTitle';
+
+        // Get source data for the ElementID field
+        $anchorOptions = function ($pageID) {
```

Comment:
For readability, semantics and debug-ability, this should be an actual method, instead of an anonymous function.